<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventana1</class>
 <widget class="QWidget" name="ventana1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>400</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ryu">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="kent">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="sold">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="killer">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="volver">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
  </widget>
  <widget class="QLabel" name="gif1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gif2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gif3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gif4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>volver</zorder>
  <zorder>gif1</zorder>
  <zorder>gif3</zorder>
  <zorder>gif4</zorder>
  <zorder>gif2</zorder>
  <zorder>kent</zorder>
  <zorder>ryu</zorder>
  <zorder>killer</zorder>
  <zorder>sold</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
